src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -44,7 +44,7 @@
  * <li>groups static imports: ensures that static imports are at the top (or the
  * bottom) of all the imports, or above (or under) each group, or are treated
  * like non static imports (@see {@link ImportOrderOption}</li>
- * </ul>
+ * </ul>.
```

Comment:
CIs were red. Checkstyle inspection for checkstyle files was failing. `SummaryJavadoc` was failing. `SummaryJavadoc` [has ties with](https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java#L175) `htmlElement` rule. Structure for this file changed with the update to grammar and `SummaryJavadoc` was detecting missed period for javadoc paragraph. I will upload regression reports for `SummaryJavadoc`.